test/bid.spec.js

```diff
@@ -916,4 +915,213 @@ describe('Bid bidObject', function() {
             done();
         });
     });
+
+    describe('#getPbBid', function() {
+
+        it('is a function', function(done) {
+            bidObject.getPbBid.should.be.a('function')
+            done();
+        });
+
+        it('returns gross cpm', function(done) {
+            expect(bidObject.getPbBid()).to.be.equal(undefined);
+            done();
+        });
+    });
+
+    describe('#setPbBid', function() {
+
+        it('is a function', function(done) {
+            bidObject.setPbBid.should.be.a('function')
+            done();
+        });
+
+        it('should set prebid bid in pBBid', function(done) {
+            expect(bidObject.pbbid).to.equal(undefined);
+            bidObject.setPbBid({"test":"bid"}).should.deep.equal(bidObject);
+            expect(bidObject.pbbid).to.deep.equal({"test":"bid"});
+            done();
+        });
+    });
+
+    describe('#getVastXml', function() {
+
+        it('is a function', function(done) {
+            bidObject.getVastXml.should.be.a('function')
+            done();
+        });
+
+        it('returns vast xml', function(done) {
```

Comment:
add a positive test case where the expected value is returned

Reply:
Done.